array_methods_lab.js

```diff
@@ -0,0 +1,255 @@
+console.log("Question 1");
```

Comment:
Great job . Seems that you understand the array method concepts. Remember to create functions both es5 or es6 is fine.